### PR TITLE
Add maintainers and governance files

### PR DESCRIPTION
**Pull request by flomonster**

### CODE REVIEW

**Review by loic-hamelin — APPROVED** [review]
lgtm

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+**3.5. Written procedures**. This governance document and other materials documenting this project's development process shall be available to any interested person.
+
+## 4. No Confidentiality.
+
+Information disclosed in connection with any Project activity, including but not limited to meetings, contributions, and submissions, is not confidential, regardless of any markings or statements to the contrary.
+
+## 5. Amendments.
+
+Amendments to this governance policy may be made by affirmative vote of 2/3 of all Maintainers, with approval by project admins.
```
Comment (SergeCroise) [inline GOVERNANCE.md:37]: just a little question:
why only 2/3 of all...,
why not all Maintainers (or e.g., 83%, 89%, or 97% of all) ?

**Review by emersion — APPROVED** [review]

File: GOVERNANCE.md (hunk 1 above)
Comment (emersion) [reply GOVERNANCE.md:37]: This comes from MVG: https://github.com/github/MVG/blob/main/project-docs/GOVERNANCE.md#6-amendments

Comment (flomonster) [reply GOVERNANCE.md:37]: It comes from the MVG [here](https://github.com/github/MVG/blob/main/project-docs/GOVERNANCE.md). The idea is to allow modifications a small group of maintainers agree without having to check with everyone.

Comment (SergeCroise) [reply GOVERNANCE.md:37]: Ok, 2/3 is a minimum according to MVG. Do not know how important amendments or governance is, but the idea is:
for the first version of this governance file, we could use 
**97% of all Maintainers**,
and later (as needed) if all Maintainers agree, we could reduce the rate to **67%** (2/3), or e.g.,
71%, 73%, 79%, 83%,...

```suggestion
Amendments to this governance policy may be made by affirmative vote of 97% of all Maintainers, with approval by project admins.
```

**Review by SergeCroise — APPROVED** [review]

File: GOVERNANCE.md (hunk 1 above)
Comment (flomonster) [reply GOVERNANCE.md:37]: I don't think that's a critical point. I prefer to have a static reasonable percentage rather than changing it when needed.

**Review by bloussou — APPROVED** [review]
Thanks

**Pull request merged**